stdlib/gzip.pyi

```diff
@@ -12,8 +12,8 @@ _ReadBinaryMode: TypeAlias = Literal["r", "rb"]
 _WriteBinaryMode: TypeAlias = Literal["a", "ab", "w", "wb", "x", "xb"]
 _OpenTextMode: TypeAlias = Literal["rt", "at", "wt", "xt"]
 
-READ: Literal[1]  # undocumented
-WRITE: Literal[2]  # undocumented
+READ: object  # undocumented
+WRITE: object  # undocumented
```

Comment:
We also need to change the type of the `GzipFile.mode` attribute on line 89 below. I wouldn't be opposed to adding the new exact values (rb and wb here) instead of using `object`.

Reply:
Thanks, fixed

I've kept it as object since it makes it more obvious if your code would fail cross-version (almost all users only type check on one version) / since CPython is clearly treating it as implementation detail that has changed in practice